app/views/declaration/carrier_eori_number.scala.html

```diff
@@ -34,6 +34,8 @@
     errorSummary: errorSummary,
     sectionHeader: sectionHeader,
     heading: heading,
+    insetText: exportsInsetText,
+    paragraphBody: paragraphBody,
     body: paragraphBody,
```

Comment:
Why 2 paragraphBody instances? You only need to specify it one time.

Reply:
Arghh, didn't notice the body: paragraphBody... :(

Reply:
Fixed